src/install-elixir

```diff
@@ -2,9 +2,23 @@
 
 set -eo pipefail
 
-cd $RUNNER_TEMP
+cd "$RUNNER_TEMP"
 
-wget -q https://repo.hex.pm/builds/elixir/${1}${2}.zip
-unzip -d .setup-elixir/elixir ${1}${2}.zip
-rm ${1}${2}.zip
-echo "$(pwd)/.setup-elixir/elixir/bin" >> $GITHUB_PATH
+VSN=${1}
+FILE_INPUT="${VSN}.zip"
+FILE_OUTPUT=elixir.zip
+DIR_FOR_BIN=.setup-elixir/elixir
+
+rm -f "${FILE_OUTPUT}"
+rm -rf "${DIR_FOR_BIN}"
+wget -q -O "${FILE_OUTPUT}" "https://repo.hex.pm/builds/elixir/${FILE_INPUT}"
+mkdir -p "${DIR_FOR_BIN}"
+unzip -q -d "${DIR_FOR_BIN}" "${FILE_OUTPUT}"
+rm -f "${FILE_OUTPUT}"
+echo "$(pwd)/${DIR_FOR_BIN}/bin" >> "$GITHUB_PATH"
+echo "Where is Erlang/OTP?"
+which erl
+echo "Installed Erlang/OTP version follows"
+$(which erl) -version
```

Comment:
We shouldn't do the erlang checks here since `install-otp` do them.

Reply:
I understand that, but since it bit me, I preferred to be conservative. Let's say the path is badly manipulated in the meantime (as was the case during development).

Reply:
eb8fb56: done. I prefer to move forward than to see the pull request declined because of this detail. If it bites us again, in the future, we'll just deal with that.